src/components/Card/Card.scss

```diff
@@ -0,0 +1,14 @@
+@import "styles/colors.scss";
+
+.card {
+  color: $color-off-white;
+  font-family: "Open sans";
+
+  position: relative;
+  z-index: 2;
```

Comment:
Can be done in tailwind, but should it be?

Reply:
yep, resolved